src/ducks/account/helpers.spec.js

```diff
@@ -426,3 +427,39 @@ describe('buildOthersReimbursementsVirtualAccount', () => {
     )
   })
 })
+
+describe('getAccountInstitutionSlug', () => {
+  const updatedApps = [
+    {
+      date: '2021-02-18T16:28:46.247Z',
+      slug: 'maif-nestor'
+    },
+    {
+      date: '2021-02-18T16:35:54.125Z',
+      slug: 'boursorama83'
+    },
+    {
+      date: '2021-02-18T16:30:46.247Z',
+      slug: 'ce'
+    }
+  ]
+  const account = updatedByApps => ({
```

Comment:
Nit: makeAccount maybe.

Reply:
Nit: I personnally like the setup method to have several tests based on the same logic.

const setup = ({ updatedByApps }) => {
  const account = { ... }
  const slug = getAccountInstitutionSlug(account)
  return { slug }
}

it('', () => {
  const { slug } = setup({ updatedByApps: undefined })
  expect(slug).toBe()
})